module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -561,6 +561,10 @@ public function getHolding($bibId, array $patron = null, array $options = [])
                 $locationData = $this->getLocationData($locationId);
                 $locationName = $locationData['name'];
                 $locationCode = $locationData['code'];
+                $holdingCallNumber = $holding->callNumber ?? '';
```

Comment:
I think you could actually move the two holding-related variable declarations above the foreach loop that's iterating over items in order to avoid a little bit of redundant work.

Reply:
:facepalm: Yes, of course. This is done.